### DISCUSSION TITLE
Custom Feature Combo Request

### DISCUSSION THREAD

**Discussion by JK-Rolling** [JK-Rolling/DuinoI2C_ESP#2] [General]
these are the example list of feature. You may request custom feature combo by referring this template.
|Feature|Value|
|---|---|
|SERIAL|Yes, 500Kbps|
|WEB_SERIAL|No|
|DASHBOARD|No|
|OLED|Yes|
|mDNS|No|
|\<others\>|\<value\>|
|ESP|Wemos Lolin D1 R2 and mini|

**Comment by JK-Rolling** [comment]
Referencing to original [request](https://github.com/JK-Rolling/DuinoI2C_ESP/discussions/1#discussioncomment-10717762) for ESP to drive external LED on D1 mini pin D7.
please get the updated firmware from [DuinoI2C_ESP.ino.d1_mini.400K.ledD7.v0.21.1.bin.gz](https://github.com/user-attachments/files/17319946/DuinoI2C_ESP.ino.d1_mini.400K.ledD7.v0.21.1.bin.gz)

**Reply by JK-Rolling** [reply]
v0.22 
[DuinoI2C_ESP.ino.d1_mini.400K.ledD7.v0.22.1.bin.gz](https://github.com/user-attachments/files/17424282/DuinoI2C_ESP.ino.d1_mini.400K.ledD7.v0.22.1.bin.gz)

**Reply by JK-Rolling** [reply]
v0.23 [DuinoI2C_ESP.ino.d1_mini.400K.ledD7.v0.23.1.bin.gz](https://github.com/user-attachments/files/17445631/DuinoI2C_ESP.ino.d1_mini.400K.ledD7.v0.23.1.bin.gz)

**Reply by JK-Rolling** [reply]
v0.24 
[DuinoI2C_ESP.ino.d1_mini.400K.ledD7.v0.24.1.bin.gz](https://github.com/user-attachments/files/17490125/DuinoI2C_ESP.ino.d1_mini.400K.ledD7.v0.24.1.bin.gz)

**Reply by JK-Rolling** [reply]
beginning in v0.25, user may choose which GPIO for external LED

**Comment by DanHKfree97** [comment]
Would it be possible to add support for this board – [ESP8266 OLED HW-364A](https://github.com/peff74/esp8266_OLED_HW-364A)? Otherwise, user-configurable I2C GPIO would be great!

**Reply by JK-Rolling** [reply]
[DuinoI2C_ESP.ino.generic.v0.28.sda14scl12.bin.gz](https://github.com/user-attachments/files/19514540/DuinoI2C_ESP.ino.generic.v0.28.sda14scl12.bin.gz)

attached custom bin. please give it a try

during early feature survey, configurable i2c pin was one of feature in the list, but no one voted for it so this feature did not get implemented. so here we are, support is still possible via this custom feature request page.
